tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,532 @@
+    if (ip_type == 'ipv4') : 
+        ixnetwork.info('Configure IPv4')
+        ipv4 = ethernet.Ipv4.add(Name='Ipv4')
+        ipv4.Address.Increment(start_value=ip_start, step_value=ip_incr_step)
+        ipv4.Address.Steps.Step = ip_incr_step
+    
+        ipv4.GatewayIp.Increment(start_value=gw_start, step_value=gw_incr_step)
+        ipv4.GatewayIp.Steps.Step = gw_incr_step
+    elif (ip_type == 'ipv6') :
+        # ipv6 stack option is left for future extension. 
+        pass
+    else :
+        logger.info("Unsupported address-type")  
+        pytest_assert(0) 
+    
+    return topology
+
+
+def start_protocols(session):
+    """This function starts all the protocols configured on the IxNetwork
+       protocol stack (e.g., IP and Ethernet).
+       
+    Args:
+        session (obj) : IxNetwork session object.
+     
+    Returns:
+        None    
+    """ 
+    ixnetwork = session.Ixnetwork
+    ixnetwork.StartAllProtocols(Arg1='sync')
+    protocolSummary = session.StatViewAssistant('Protocols Summary')
+    protocolSummary.CheckCondition('Sessions Not Started', protocolSummary.EQUAL, 0)
+    protocolSummary.CheckCondition('Sessions Down', protocolSummary.EQUAL, 0)
+    logger.info(protocolSummary)
+
+
+def stop_protocols(session) :
+    """This function stops all the protocols configured on the IxNetwork
+       protocol stack (e.g., IP and Ethernet).
+
+    Args:
+        session (obj) : IxNetwork session object.
+
+    Returns:
+        None
+    """
+    ixnetwork = session.Ixnetwork
+    ixnetwork.StopAllProtocols(Arg1='sync')
+
+
+def get_traffic_statistics(session, stat_view_name='Flow Statistics'):
+    """This function fetches the traffic statistics information.
+       
+    Args:
+        session (obj) : IxNetwork session object.
+        stat_view_name (str, optional): Statistics view name. Default 
+            value is 'Flow Statistics'
+
+    Returns:
+        traffic statistics dictionary. 
+    """
+    ixnetwork = session.Ixnetwork
+    traffic_statistics = session.StatViewAssistant(stat_view_name)
+    ixnetwork.info('{}\n'.format(traffic_statistics))
+    return traffic_statistics
+
+
+def stop_traffic(session):
+    """ This function stops all the IxNetwork traffic items configured 
+        on all the ports.
+    Args:
+        session (obj): IxNetwork session object.
+
+    Returns:
+        None.   
+    """
+    ixnetwork = session.Ixnetwork
+    ixnetwork.Traffic.StopStatelessTrafficBlocking()
+
+
+def start_traffic(session):
+    """ This function starts all the IxNetwork traffic items configured 
+        on all the ports.
+    Args:
+        session (obj): IxNetwork session object.
+
+    Returns:
+        None.   
+    """
+    ixnetwork = session.Ixnetwork
+    """ Apply traffic to hardware """
+    ixnetwork.Traffic.Apply()
+    """ Run traffic """
+    ixnetwork.Traffic.StartStatelessTrafficBlocking()
+
+
+def create_ip_traffic_item_using_wizard_arguments (
```

Comment:
just 'create_ip_traffic_item'  should be good

Reply:
Changed the name. create_ip_traffic_item_using_wizard_arguments -> create_ip_traffic_item